gdtf.py

```diff
@@ -262,11 +262,10 @@ def loadModel(profile, model):
         if obj.dimensions.z <= 0:
             DMX_Log.log.error(f"Model {obj.name} Z size {obj.dimensions.z} <= 0. It will likely not work correctly.")
 
-        dim_x = obj.dimensions.x or 1
-        dim_y = obj.dimensions.y or 1
-        dim_z = obj.dimensions.z or 1
-
-        obj.scale = (obj.scale.x * model.length / dim_x, obj.scale.y * model.width / dim_y, obj.scale.z * model.height / dim_z)
+        obj.rotation_mode = 'XYZ'
+        scale_vector = obj.scale * obj_dimension
+        dimensions = obj.dimensions or Vector((1, 1, 1))
```

Comment:
This will actually fail if any of the dimensions (x, y, z) is 0 or below zero.

Reply:
@vanous  I see, but dimensions can never be negative, only zero would be a problem, I made another PR wich catches that problem:  7cd9197d1d1112db89cfdefb8da2cbc7f8914502